File: Codeforces/Golang/800/155A -  I_love_%username%/main.go
package main

import (
	"fmt"
)

func main() {
	var n, count int
	fmt.Scan(&n)

	var score int
	fmt.Scan(&score)

	max := score
	min := score

	for i := 1; i < n; i++ {
		fmt.Scan(&score)
		if score > max {
			max = score
			count++
		} else if score < min {
			min = score
			count++
		}
	}
	fmt.Println(count)
}

File: Codeforces/Rust/800/155A -  I_love_%username%/src/main.rs
use std::io;

fn main() {
    let mut input = String::new();
    let _ = io::stdin().read_line(&mut input);

    let n: usize = input.trim().parse().unwrap();

    let mut scores_input = String::new();
    let _ = io::stdin().read_line(&mut scores_input);

    let score: Vec<i64> = scores_input
        .split_whitespace()
        .map(|s| s.parse().unwrap())
        .collect();

    let mut max = score[0];
    let mut min = score[0];
    let mut count = 0;

    for i in 1..n {
        if score[i] > max {
            max = score[i];
            count += 1
        } else if score[i] < min {
            min = score[i];
            count += 1
        }
    }
    println!("{}", count)
}
